.github/ISSUE_TEMPLATE/03_docs_update.yml

```diff
@@ -0,0 +1,40 @@
+name:  Updates or Enhancements for Documentation
+description: Documents out of date?  Examples need reworking? Got an idea for a new article?
+title: "Documentation:  <Insert Issue Title Here>"
+labels: ["vignettes", "website", "documentation"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        The admiral team loves documentation!  Help us keep it up to date and fresh!  
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: Please select a category the issue is focused on?
+      multiple: true
+      options:
+        - Function Examples in Documentation
+        - Function Documentation/Roxygen Headers
+        - Get Started Document
+        - Developer Guides
+        - User Guides
+        - I have a new documentation idea! Yay!
+  - type: textarea
+    id: docs-out-if-sync
+    attributes:
+      label: Let us know where something needs a refresh or put your idea here!
+      description: Screen shots are super helpful here!  You can do a screen grab and cut and paste it here right in here.
```

Comment:
Should it be "Screenshots" to be consistent?

Reply:
Updated to Screenshot